march_hardware/include/march_hardware/Joint.h

```diff
@@ -49,10 +49,12 @@ class Joint
 
   void actuateRad(double target_position);
   void actuateTorque(int16_t target_torque);
+  void readEncoders(const ros::Duration& elapsed_time);
 
-  double getAngleRadAbsolute();
-  double getAngleRadIncremental();
-  double getAngleRadMostPrecise();
+  double getPosition();
+  double getVelocity();
+  double getIncremental_position();
+  double getAbsolute_position();
```

Comment:
You should make these methods `const`, since all they do is return a value without modifying the class. (Also use camelCase naming :stuck_out_tongue:)
```suggestion
  double getPosition() const;
  double getVelocity() const;
  double getIncrementalPosition() const;
  double getAbsolutePosition() const;
```

Reply:
Oof to think that name slipped through :fearful: